samples/install-without-bom/pom.xml

```diff
@@ -32,7 +32,7 @@
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud-spanner</artifactId>
-      <version>6.21.2</version>
+      <version>6.23.2</version>
```

Comment:
This is already part of this PR: https://github.com/googleapis/java-spanner/pull/1788/files